scripts/cover.sh

```diff
@@ -82,10 +71,11 @@ for pkg in "$@"; do
 		args="-coverprofile $COVER/cover.${i}.out -covermode=atomic -coverpkg $coverpkg"
 	fi
 
-	do_waitpid go test -race $args "$pkg" 2>&1 \
-		| grep -v 'warning: no packages being tested depend on'
+  echo " - go test -race ${args} \"${pkg}\"" >> "${commands_file}"
 done
-reset_waitpids
+parallel-exec --fast-fail --no-log --max-concurrent-cmds 4 --dir "${DIR}" "${commands_file}" 2>&1 \
```

Comment:
No logs here is really overkill, I actually want to reduce the noise here, it's a long story.